### DISCUSSION TITLE
Lay-Z-Spa Palm Springs (S100101)

### DISCUSSION THREAD

**Discussion by MeisterTR** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#212] [Show and tell]
First i have to say, this is a very cool project.
Since last year i have a Palm Springs with square Pump (S100101) it takes some days to figure out witch resistors i need. This model seems a bit diiferent to the S100103 where 6 resistors needed with 560 Ohms.  I only need 3 resistors to the Dislplay, wehen i also put 3 to the pump, the display  isnt working anymore.

I hope this will help someone with the same problem
![20220213_174908](https://user-images.githubusercontent.com/6328382/153765808-4c191dbc-bc83-412b-8696-6b2f19a3d963.jpg)

![20220213_174923](https://user-images.githubusercontent.com/6328382/153765811-b5b34c24-1bfa-42a3-bf71-3bfd3680cdf0.jpg)

**Comment by visualapproach** [comment]
Awesome box 🥇 !
Thank you for sharing your findings here, @MeisterTR

**Comment by Skeletorjus** [comment]
@MeisterTR, what issues did you experience? I have a S100101 (Miami 2021), and during my short dry test runs everything seemed fine.

**Reply by eried** [reply]
Did you succeed @Skeletorjus ? I have a Tahiti but the same pump 🤓 but I haven't had any success yet, just waiting on parts now to test more stuff

**Reply by Skeletorjus** [reply]
Yes, worked on first try, no resistors. I do have a minor issue with momentary wrong states (https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/issues/236), but it doesn't really affect anything.
I've put together a second board with shorter cables to see if it helps, but haven't had a chance to make the switch yet.

**Reply by eried** [reply]
Nice. I have no idea why it doesn't work on mine 😵

**Reply by Skeletorjus** [reply]
What errors are you experiencing? Have you tested your connections with a multimeter?

**Reply by eried** [reply]
With a multimeter? you mean to see if there is conductivity? (yes I did, also with a digital analyser)

On the first attempt, when I connected to my S100101 I hear the boot music, and in the webpage I get the current temperature but the screen never turns on or any of the actions work (pump, heater).

What I have already tried:
1) Adding resistors (560) 
2) Add pull downs with caps
3) Recreate the whole pcb with all new parts 
4) Shorter cables

Next I will assemble another solution without the 8 channel level converter, but 2 individual 4 channel ones.
